domain/camera/src/test/java/com/google/jetpackcamera/domain/camera/test/FakeCameraUseCaseTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023 The Android Open Source Project
+ * Copyright (C) 2023-2024 The Android Open Source Project
```

Comment:
Replace with just 2024

Reply:
This is spotless's doing. I'm going to turn off this behavior in a separate PR and we'll stick with the inception year in each file (as is normally done in Android).